src/BundlerMinifier.Core/BundlerMinifier.Core.csproj

```diff
@@ -5,21 +5,19 @@
   <PropertyGroup>
     <TargetFrameworks>net46;netstandard2.0;netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
     <DefineConstants>$(DefineConstants);DOTNET</DefineConstants>
-    <Authors>Mads Kristensen, salar2k</Authors>
+    <Authors>Argo Zhang</Authors>
```

Comment:
We should never remove the original author!

Reply:
sorry for that. I want to publish my own package, so I changed the author. this update auto update this PR. My project cannot be compiled because your package has not been updated. Thank you for updating the package